tests/Linting/Linters/ImportFacadesTest.php

```diff
@@ -52,6 +52,27 @@ function does_not_trigger_on_facade_usage_with_import()
 
 use Illuminate\Support\Facades\Hash;
 
+var_dump(Hash::make('test'));
```

Comment:
I think we can probably safely make this code sample for the test more minimal, e.g. remove var_dump & lines 58-62, but the logic of the tests looks good 👌